Create_HighScores_Table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE HighScores (
+ID INT NOT NULL IDENTITY(1,1) PRIMARY KEY,
+GameType VARCHAR(13) NOT NULL CHECK (GameType IN('Beginner', 'Easy', 'Medium', 'Hard', 'DBTest')),

```

Comment:
Actually I would probably use the INT instead of the string........... may not have time though